Source/sound.cpp

```diff
@@ -227,9 +242,11 @@ void music_start(uint8_t nTrack)
 			if (musicRw == nullptr)
 				ErrSdl();
 #endif
-			music = Mix_LoadMUSType_RW(musicRw, MUS_NONE, /*freesrc=*/1);
-			if (music == nullptr) {
-				Log("Mix_LoadMUSType_RW: {}", Mix_GetError());
+			music.emplace(musicRw, std::make_unique<Aulib::DecoderDrwav>(),
+			    std::make_unique<Aulib::ResamplerSpeex>(sgOptions.Audio.nResamplingQuality), /*closeRw=*/true);
+			if (!music->open()) {
+				SDL_LogError(SDL_LOG_CATEGORY_AUDIO, "Aulib::Stream::open (from music_start): %s", SDL_GetError());

```

Comment:
```suggestion
				LogError(Audio, "Aulib::Stream::open (from music_start): %s", SDL_GetError());
```

Reply:
Done

Reply:
I think there are lots of them.

Reply:
Don't think I've missed, have I?

Reply:
Oh, i thought you just changed that one, maybe stale diff on my end.